### PR TITLE
[Feat] Grouped Bar Charts

### PR DESCRIPTION
**Pull request by speckospock**
Adds a new React wrapper for the Grouped Bar Chart component

## Description

A new GroupedBar component has been added to the components, following the style and test coverage of existing components

## Motivation and Context

Takes a step towards 1:1 feature parity with Britecharts and Britecharts-React

## How Has This Been Tested?

Per the Contributing Guide, testing was performed by adding test coverage modified from the StackedArea test coverage step-by-step, then checked with the documentation for proper rendering.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,16 @@
...
+Import components from Britecharts-React:
+
+```js static
+//ES6 import syntax
+import { StackedArea } from 'britecharts-react';
+
+//CommonJS require syntax
+const { StackedArea } = require('britecharts-react');
+```
+
```
Comment (Golodhros) [inline README.md]: Makes sense, thanks a lot!

File: README.md [hunk 2]
```diff
@@ -37,6 +47,21 @@ From there, we will proceed to pass each of the configurations through the usual
...
+- Stacked Area charts [(API)][stackedAreaChartAPI]
+- Tooltips [(API)][tooltipAPI]
+- Loading states
+
+The following components are not yet implemented:
+- Brush charts
+- Stacked Bar charts
+- Sparkline charts
+- Step charts
+
```
Comment (Golodhros) [inline README.md]: Thanks for this!!

**Comment by albybarber** [comment]
@miglesiasEB This looks sweet! Let's merge it in!

**Comment by speckospock** [comment]
Thanks all! Glad to be of help.

**Comment by Golodhros** [comment]
You definitely have been of help @speckospock! We really appreciate your contribution!

Don't hesitate to give us feedback about the onboarding, especially if you think we can improve anything in our docs!

**Pull request merged**